Project 4/README.md

```diff
@@ -45,3 +45,133 @@ You're probably thinking, "why would I submit a level 8 kata if they're not wort
 It is your responsibility and the responsibility of your peers reviewing your submission in PR to determine whether your submission is ranked appropriately. In the event that consensus is reached that your kata is ranked inappropriately, you must work with your peers to revise the submission so that it is either more or less challenging, accordingly. You are not permitted to submit new problems with different strengths after PRs are open, but must instead revise your PRs. So, think hard about how challenging your submission is. 
 
 There is one other option for those desiring a different sort of challenge. If you provide alongside your SPARQL submission a translation of the same problem into SQL, complete with documentations, solution, etc. then you may receive half points extra at that kata level (rounded up). For example, if you submit a SPARQL problem that is kata rank 1 and also submit a SQL version of that same problem, you  will receive 35+18=53 points. 
+
+My Best Attempts.....
+Kata Level 6 – 3 point
+All actors who have played the character of Batmen.
+PREFIX dbo: http://dbpedie.org/ontology/
+PREFIX dbr: http://dbpedia.org/research/
+Select Distinct ?actor WHERE {
+?film dbo:starring ?act .
+?film dbo:wikiPageRedircts dbr:Batman_(film_series) . 
+?actor dbo:wikiPageID ?id . 
+FILTER (REGEX(STR(?id), “Batman_films_casts”, “I”))
```

Comment:
I chose starring ?act because there could be multiple actors in a given project but the starring act would be the star of the film. I could change it to lead ?act. Additionally, I am hesitant to use actor as there is a big push in hollywood to call all actors actors -- not discriminate between male/non-binary/female etc..  so the starring act can account for this shift.... I hope lol

Reply:
Oh yeah you can definitely use the term you believe fits best in this situation (sorry for not recognising what was going on), my issue was that you probably want it to be the same term you then use in line 57, where you wrote ?actor, since it seems you want to refer to the same entity